src/bitbutler.sh

```diff
@@ -698,6 +701,69 @@ JSON
   esac
 }
 
+function selfupdate() {
+  REPOSITORY_SLUG="particleflux/bitbutler"
+  SELFUPDATE_WORKING_DIR='/tmp/bitbutler-selfupdate'
+  SETTINGS_FILE="$BB_VENDOR_PATH/make.settings"
+
+  latest_tag_json=$(curl \
+    --silent \
+    --header "Accept: application/vnd.github.v3+json" \
+    "https://api.github.com/repos/${REPOSITORY_SLUG}/releases" |
+    jq '.[0]')
+
+  if [[ "$latest_tag_json" == "null" ]]; then
+    die "No releases yet, cannot update"
+  fi
+
+  latest_tag_name=$(jq --raw-output '.tag_name' <<<"$latest_tag_json")
+
+  v "Current script version: ${SCRIPT_VERSION}"
+  v "Latest version is: $latest_tag_name"
+  if [[ "$latest_tag_name" != "${SCRIPT_VERSION}" ]]; then
+    l "Your version is outdated."
+  else
+    l "You already have the lasted version."
```

Comment:
Typo: `lasted` vs `latest`

Reply:
The latest version will last a long time. :D
No idea what kind of brain-autocorrect that was^^